_schemaV2.graphql

```diff
@@ -365,6 +365,14 @@ enum AlertSettingsFrequency {
   INSTANT
 }
 
+input AlertSettingsInput {
+  details: String
+  email: Boolean
+  frequency: AlertSettingsFrequency
+  name: String
```

Comment:
**thought (non-blocking):** I suppose it's fine to keep `name` for parity with the old implementation, but noting that we've [dropped](https://github.com/artsy/force/pull/13252) custom names from the UI. 

(We always have the caveat about old Eigen versions, but those would continue to use the old schema, so I don't think that's a factor here)

Reply:
It's also available via the new API! In https://github.com/artsy/gravity/blob/6b58d4dfa7553c94aa83c9d44e48a463cd9496f8/app/api/v1/me_alerts_endpoint.rb#L12

I would be fine to remove that later. Any reason why that wasn't removed from the stitched-in version, maybe to keep it accessible?

Reply:
I think there is no harm in keeping this although less than `2%` of our users are using a version where the name has not been removed (8.28.0). 
<img width="1325" alt="Screenshot 2024-01-09 at 10 34 28" src="https://github.com/artsy/metaphysics/assets/11945712/d17b7ac1-bf6f-403f-808d-4ec414c91ef1">